edb/edgeql-parser/edgeql-parser-python/src/normalize.rs

```diff
@@ -159,9 +159,10 @@ pub fn normalize(text: &str) -> Result<Entry, Error> {
                 matches!(kw, "configure"|"create"|"alter"|"drop"|"start"|"analyze")
                 || (last_was_set && kw == "global")
             ) => {
+                let processed_source = serialize_tokens(&tokens);
```

Comment:
Please add a comment here why this is, indeed, needed.

Reply:
*This* one probably isn't really, and I could drop it if you want. (This case is when we are bailing because it is DDL or some such, and can't hit a cache anyway)

I can add a comment to the other case

Reply:
Added